### ISSUE TITLE
Can't join ~dopzod/urbit-help

### ISSUE THREAD

**Issue by mattlevan** [urbit/urbit#1695]
After re-booting my planet `~mastyr-bottec` with the same key as before, I can't join `~dopzod/urbit-help`, although I receive "successful" messages. I can't see the circle in Landscape and, even though the circle is bound to a rune in the CLI, when I run `;who` or `;circles`, there's no indication that I'm a member of it.

![image](https://user-images.githubusercontent.com/4582375/64302088-9c5c7780-cf36-11e9-99a2-f23d8ec4ed6d.png)

**Comment by vvisigoth** [comment]
I'm confused. It looks like you've successsfully joined urbit help from that last line. Is the issue that you can't see it in the UI?

**Comment by mattlevan** [comment]
Sorry it isn't clear. It says success but I can't see any messages on CLI or Landscape. The rune gets bound and I can switch to it but nothing happens. I don't see any messages and can't send any either. Running `;circles` doesn't list the subscription either.

**Comment by vvisigoth** [comment]
That is truly odd. I can |hi ~mastyr, but ~dopzod can not. We haven't seen
any OWC yet this era, so I'm interested to know if this is a case of one.

A
—
~poldec-tonteg
http://urbit.org

**Comment by mattlevan** [comment]
I can hi ~dopzod successfully!

**Comment by vvisigoth** [comment]
Can you try this

> |start %eth-manage
> :eth-manage %look-ethnode

I wonder if someone here doesn't have the latest keys
—
~poldec-tonteg
http://urbit.org

**Comment by mattlevan** [comment]
Sure.

Looks like I get lots of `; ship has sunk` and `%never-heard-of-her` messages. Joe described some process by which I should update my key, but I'm not sure how to go about it. What do I need to do (if that's the solution here)?

**Comment by vvisigoth** [comment]
Interesting, it looks like ~wanzod can't "hi" you either. OK, basic stuff:
what version of urbit are you on?

```
; ~mastyr-bottec not responding still trying
; ~mastyr-bottec is ok
; ~mastyr-bottec not responding still trying
; ~mastyr-bottec is ok
; ~mastyr-bottec not responding still trying
; ~mastyr-bottec is ok
; ~mastyr-bottec not responding still trying
; ~mastyr-bottec is ok
; ~mastyr-bottec not responding still trying
; ~mastyr-bottec is ok
~wanzod:dojo>
```
—
~poldec-tonteg
http://urbit.org

**Comment by mattlevan** [comment]
v0.8.2

**Comment by matildepark** [comment]
I was on a Discord call with @mattlevan sorting this out. It seems like they might've possibly rebooted their ship from scratch within the same continuity era (eg. `./urbit mastyr-bottec -w -k`), from their description. They could join and chat in ~paldev/nopub without issue, it's just that their sponsor and ~dopzod didn't want anything to do with them.

It therefore seemed like a good candidate for a personal breach. We now might want ~wanzod to update their keys for them to get them going smoothly.

**Comment by vvisigoth** [comment]
I'll kick.
—
~poldec-tonteg
http://urbit.org

**Comment by mattlevan** [comment]
Yep, so I performed a personal breach (reset networking keys) thanks to the help of @matildepark. Just joined `~dopzod/urbit-help` again and it seems to be loading the messages just fine (`ames` has been spinning for awhile). I received the new Landscape over the air as well (since I can see the "Home" splash message in Chat).

**Comment by vvisigoth** [comment]
Nice!
—
~poldec-tonteg
http://urbit.org
